src/v/kafka/group_probe.h

```diff
@@ -82,10 +83,31 @@ class group_offset_probe {
              sm::description("Consumer group committed offset"),
              labels)
              .aggregate({sm::shard_label})});
+
+        _public_metrics.add_group(
+          prometheus_sanitize::metrics_name("kafka:consumer:group"),
+          {sm::make_gauge(
+             "committed_offset_timestamp_seconds",
+             [this] { return _commit_timestamp.value()/1000; },
+             sm::description("Consumer group commit offset timestamp"),
+             labels)
+             .aggregate({sm::shard_label})});
+
+        _public_metrics.add_group(
+          prometheus_sanitize::metrics_name("kafka:consumer:group"),
+          {sm::make_gauge(
+             "committed_offset_age_seconds",
+             [this] {
+               return (long)(model::timestamp::now().value() - _commit_timestamp.value())/1000;
+             },
+             sm::description("Consumer group consumer offset age"),
+             labels)
+             .aggregate({sm::shard_label})});
```

Comment:
Since this can be calculated from `committed_offset_timestamp_seconds`, I think it's best to leave that to the time series database, and not duplicate the data.